pkg/server/router/manifest.go

```diff
@@ -44,14 +44,38 @@ func NewManifestRouter(s svcframework.Service) (*ManifestRouter, error) {
 // CreateManifestRequest is the request body for creating a manifest, which populates all remaining fields
 // and builds a well-formed manifest object.
 type CreateManifestRequest struct {
-	Name                   *string                          `json:"name,omitempty"`
-	Description            *string                          `json:"description,omitempty"`
-	IssuerDID              string                           `json:"issuerDid" validate:"required"`
-	IssuerKID              string                           `json:"issuerKid" validate:"required"`
-	IssuerName             *string                          `json:"issuerName,omitempty"`
-	ClaimFormat            *exchange.ClaimFormat            `json:"format" validate:"required,dive"`
-	OutputDescriptors      []manifestsdk.OutputDescriptor   `json:"outputDescriptors" validate:"required,dive"`
-	PresentationDefinition *exchange.PresentationDefinition `json:"presentationDefinition,omitempty" validate:"omitempty,dive"`
+	// Summarizing title for the Manifest in question.
+	// Optional.
+	Name *string `json:"name,omitempty"`
+
+	// Explains what the Manifest in question is generally offering in exchange for meeting its requirements.
+	// Optional.
+	Description *string `json:"description,omitempty"`
+
+	// DID that identifies who the issuer of the credential(s) will be.
+	// Required.
+	IssuerDID string `json:"issuerDid" validate:"required"`
+
+	// The KID of the private key that will be used when signing issued credentials.
+	// Required.
+	IssuerKID string `json:"issuerKid" validate:"required"`
+
+	// Human-readable name the Issuer wishes to be recognized by.
+	// Optional.
```

Comment:
Having the optional comment is nice, but a little jank. I wonder if there is a way to specify it in the `json:` part

I guess omitempty is this, 'if its missing will not add it to the json obect' so anything that has omitempty is optional.

Its find though, I think the comment is good

Reply:
We need to be able to specify this optionality at the API level. It would be awesome if we could simply use a type and it get automatically populated, but our tooling doesn't support it :(